flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/DeclarativeSlotPoolBridgeResourceDeclarationTest.java

```diff
@@ -141,40 +137,38 @@ public void testRequirementsDecreasedOnAllocationTimeout() throws Exception {
                             .get();
 
             // waiting for the timeout
-            assertThat(
-                    allocationFuture,
-                    FlinkMatchers.futureWillCompleteExceptionally(Duration.ofMinutes(1)));
-
+            assertThatFuture(allocationFuture).failsWithin(Duration.ofMinutes(1));
+            Thread.sleep(5L);
```

Comment:
Why is it needed here?

Reply:
Nice catch! 
sorry for it. Maybe it's a dirty line from the cp.
I'll remove it and check it on the other corresponding PRs.